web/packages/teleport/src/services/localStorage/localStorage.ts

```diff
@@ -125,16 +125,24 @@ const storage = {
       return userPreferences.theme;
     }
 
-    const theme = window.localStorage.getItem(
-      KeysEnum.THEME
-    ) as DeprecatedThemeOption;
+    const theme = this.getDeprecatedThemePreference();
     if (theme) {
       return theme === 'light' ? ThemePreference.Light : ThemePreference.Dark;
     }
 
     return ThemePreference.Light;
   },
 
+  // TODO(ryan): remove in v15
```

Comment:
```suggestion
  // DELETE IN 15 (ryan)
```

Grepping for `DELETE IN` is how we typically discover these.